proto/storage/report_snapshot.proto

```diff
@@ -31,6 +31,9 @@ message ReportSnapshot {
   ReportStatus report_status = 9;
   repeated NotifierSnapshot notifiers = 10;
   SlimUser requester = 11;
+  oneof optionalColumns{
+    VulnerabilityReportOptionalColumns vulnerabilityReportOptionalColumns = 12;
+  }
```

Comment:
@ksurabhi91 @sachaudh  If I understand correctly, corresponding change to frontend code:
* replace `vulnReportFilters.includeNvdCVss` and so on
* with `vulnerabilityReportOptionalColumns.includeNvdCvss`

And other future report types might:
* either not have optional columns
* or have something like `complianceReportOptionalColumns.includeWhatever`

Reply:
Question for understanding, not change request, is there any style convention?
* `vuln_report_filters` has snake case in proto and camel case in JSON
* `vulnerabilityReportOptionalColumns` has camel case in both proto and JSON